spec/models/area_spec.rb

```diff
@@ -0,0 +1,10 @@
+describe Area, type: :model do
+  # Shoulda Callback Matchersは以下を参照
```

Comment:
__[rubocop]__
* Use only ascii symbols in comments. [:ref](https://github.com/bbatsov/ruby-style-guide#english-comments)